cloudfront-apigw-rest-api-lambda-dynamodb-sam/src/function.js

```diff
@@ -3,9 +3,11 @@
 
 const randomBytes = require('crypto').randomBytes;
 
-const AWS = require('aws-sdk');
+const { DynamoDBClient } = require('@aws-sdk/client-dynamodb');
+const { DynamoDBDocumentClient, PutCommand } = require('@aws-sdk/lib-dynamodb');
```

Comment:
note: Some error occurs. Because in Node.js 18 and later runtimes, AWS SDK v3 is bundled, so I rewrote it to v3. See articles below.

- https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/
- https://aws.amazon.com/blogs/compute/node-js-22-runtime-now-available-in-aws-lambda/